### PR TITLE
Explore page

### PR DESCRIPTION
**Pull request by abhishektanwar**
- added business logic to explore page and card slider on home page
- refactored video card

Files to check:
- src/Contexts/VideoListingContext/index.jsx
- src/Reducers/videosReducer.js
- src/components/AsideNav/index.jsx
- src/screens/ExploreVideos/index.jsx

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/components/CardSlider/index.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import BadgeIconButton from '../Buttons/BadgeIconButton'
 import './card-slider.css'
 
 const CardSlider = (props) => {
-  const {heading} = props;
+  const {heading,videos} = props;
   // const featuredVideos = videos.filter(video=>video.featured)
```
Comment (zoef-yt) [inline src/components/CardSlider/index.jsx:9]: remove commented code.

File: src/screens/History/index.jsx [hunk 2]
```diff
@@ -9,9 +9,9 @@ const History = () => {
         <h3>Watch History</h3>
         <Button buttonStyle="secondary-button" icon={<DeleteIcon />} buttonText="Clear All History" />
       </div>
-      <div className="flex-column flex-align-item-center">
+      {/* <div className="flex-column flex-align-item-center">
         {[1,2,3,4,5,6,7,8].map((item)=><VideoCard />)}
-      </div>
+      </div> */}
```
Comment (zoef-yt) [inline src/screens/History/index.jsx:14]: remove commented code.

**Review by sanu29 — APPROVED** [review]
All good to proceed with commit according to me. Great work could not find any issues.
Spacing is great as well your naming conventions are meaningful and easy.
File structure is also good. ES6 used when ever possible. Imports done right
Great work.

**Review by zoef-yt — APPROVED** [review]
The code looks good. Good work. You are good to merge.

**Pull request merged**